packages/vega-transforms/src/util/AggregateOps.js

```diff
@@ -34,7 +34,7 @@ export const AggregateOps = {
   sum: {
     init:  m => m.sum = 0,
     value: m => m.sum,
-    add:  (m, v) => m.sum += v,
+    add:  (m, v) => m.sum += +v,
     rem:  (m, v) => m.sum -= v,
```

Comment:
Why not this as well?

```suggestion
    rem:  (m, v) => m.sum -= +v,
```

Reply:
minus automatically converts strings to ints (e.g. "2" - 1 = 1)

Reply:
Nananananan https://www.destroyallsoftware.com/talks/wat